als-server/shared/src/main/scala/org/mulesoft/als/server/modules/workspace/WorkspaceContentManager.scala

```diff
@@ -205,6 +205,8 @@ class WorkspaceContentManager private (val folderUri: String,
                 mainFileUri.exists(
                   u => repository.getUnit(u).exists(_.parsedResult.result.baseUnit.isInstanceOf[ExternalFragment])) =>
             true
+          case _ if s.parsedResult.amfConfiguration.workspaceConfiguration != workspaceConfiguration =>
+            true // workspace configuration has changed since last parse
```

Comment:
is it not easier to move the `if` predicate as the return and avoid a second `case`?

Reply:
You mean something among the lines of: 
```suggestion
            case _ => s.parsedResult.amfConfiguration.workspaceConfiguration != workspaceConfiguration
```

Reply:
yess